devdocs/staticAnalysis.md

```diff
@@ -90,6 +98,8 @@ Alternatively, run the tools via Gradle:
 where `{toolType}` = checkstyle, pmd, findbugs (lowercase), and `{sourceCodeType}` = Main, Test (Pascal Case).
 The reports can be found in the `build/reports/{toolType}/` directory.
 
+To run Checkstyle analysis on all Java source files with the Eclipse Checkstyle plugin, right click on the Project Folder in the `Project Explorer` window in Eclipse and select `Checkstyle > Check Code with Checkstyle`. The report can be found in the `Markers` window in Eclipse.
```

Comment:
I'd say this has been described (although not in this great detail) two paragraphs before this. Move it there, perhaps, or just exclude this part?